src/mame/mame.lst

```diff
@@ -38107,6 +38109,7 @@ storioes                        // (c) 2011 V.Tech
 storioesa                       // (c) 2011 V.Tech
 storiofr                        // (c) 2011 V.Tech
 storionl                        // (c) 2011 V.Tech
+innotab2
```

Comment:
Please keep the entries in this file sorted in some kind of recognisable order, whether it’s alphabetical like it is for most source files, or organised by chipset like the PC BIOS holder drivers.